doc/manual/en/config/widgets/plugins/strftime/index.rst

```diff
@@ -0,0 +1,93 @@
+.. _strftime:
+
+The strftime plugin
+===================
+
+.. api-doc:: strftime
+
+Description
+-----------
+
+.. ###START-WIDGET-DESCRIPTION### Please do not change the following content. Changes will be overwritten
+
+This plugins integrates formated date and clock strings into based on strftime.
+
+Thanks to Michael Markstaller for implementing the jqclock plugin as reference.
```

Comment:
I don't think that this line belongs in a manual.

When we take that line out of the source file would that automatically be fixed here or would that require a manual process call?